src/js/widgets/CheckboxWidget.jsx

```diff
@@ -15,7 +15,9 @@ export default function CheckboxWidget({
   const widgetClasses = classNames('form-checkbox', options.widgetClassNames);
   return (
     <div className={widgetClasses}>
-      <input type="checkbox"
+      <input
+        type="checkbox"
+        autoComplete={options.autocomplete || null}
```

Comment:
Can I recommend removing the `autocomplete` attribute here? Checkboxes will throw an axe error because the attribute is intended for inputs that will take text or numeric input. I found this helpful article on MDN:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values